generated/connectors/bundled.yaml

```diff
@@ -2590,6 +2664,7 @@ components:
         - .servicenow-sir
         - .server-log
         - .slack
+        - .slack-api
```

Comment:
It doesn't look like this in the version of this spec in the [Kibana repo](https://raw.githubusercontent.com/elastic/kibana/main/x-pack/plugins/actions/docs/openapi/bundled.yaml) is it in the process of getting added there?

Reply:
It seems to be missed in that location in the OpenAPI spec, I'm reaching out to them to eventually find out what's up. I have confirmed manually that it exists in the response payload I get from Kibana.